config/initializers/monkeypatches/deliver_after_commit.rb

```diff
@@ -7,3 +7,29 @@ def deliver_after_commit
     end
   end
 end
+
+module AfterCommitEverywhereWithLocale
+  def initialize(connection: ActiveRecord::Base.connection, **handlers)
+    @connection = connection
+    @handlers = handlers
+    @locale = I18n.locale
+  end
+
+  def before_committed!(*)
+    I18n.with_locale(@locale) { @handlers[:before_commit]&.call }
+  end
+
+  def committed!(*)
+    I18n.with_locale(@locale) { @handlers[:after_commit]&.call }
+  end
+
+  def rolledback!(*)
+    I18n.with_locale(@locale) { @handlers[:after_rollback]&.call }
+  end
+end
+
+if AfterCommitEverywhere::VERSION == "1.4.0"
```

Comment:
There are some Rubocop warnings on this file. You should fix the `Layout/TrailingEmptyLines` one.

The `Rails/Output` I think is actually fine. You can update Rubocop's rules from:

https://github.com/otwcode/otwarchive/blob/8467810930a2a5f5213a62cbea0d6d031eff2fed/.rubocop.yml#L155-L158

to:

```yml
Rails/Output:
  Exclude:
    # Allow patches to print warnings to console:
    - 'config/initializers/monkeypatches/*.rb'
    # Allow migrations to print pt-osc comments to console:
    - 'db/migrate/*.rb'
```